airbyte-webapp-e2e-tests/README.md

```diff
@@ -1,5 +1,17 @@
-## Start Cypress
+## Running Cypress tests locally
+Except as noted, commands are written as if run from inside the `airbyte-webapp-e2e-tests/` directory.
 
-### `npm install`
+Prerequisites:
+1) install the e2e test dependencies and tooling with `npm install`
+2) build the OSS backend for the current commit with `SUB_BUILD=PLATFORM ../gradlew clean build`
+3) create the test database: `npm run createdb`
+4) start the OSS backend: `VERSION=dev docker-compose --file ../docker-compose.yaml up`
 
-### `npm run cypress:open`
+If you want the ability to change the frontend as you test (to add `data-testid` attributes to the components being tested, for example, or to practice test-driven development), there are a few additional steps. These are optional, but recommended:
+5) run `docker-compose stop webapp` to turn off the dockerized frontend build (which uses a production-style build instead of a hot-reloading dev server)
```

Comment:
This command returns `ERROR: Duplicate mount points: [workspace:.:rw, .:.:rw]`

Reply:
I'm guessing this is an issue with something I've done since the UI doesn't run for me unless I manually `npm start` like below.

Reply:
> This command returns ERROR: Duplicate mount points: [workspace:.:rw, .:.:rw]

You mean `docker-compose stop webapp`? Weird, I haven't seen that. What version of `docker-compose` are you using? (for reference, I'm using v2.10.0, but my setup may not be ideal for comparison: linux vs mac is a potentially big confounding factor here)

Reply:
`docker-compose version 1.29.2, build 5becea4c`
I upgraded and this works now 👍